builder/node.go

```diff
@@ -130,6 +133,7 @@ func (b *Builder) LoadNodes(ctx context.Context, opts ...LoadNodesOption) (_ []N
 					Platforms:       n.Platforms,
 					ContextPathHash: b.opts.contextPathHash,
 					DialMeta:        lno.dialMeta,
+					Timeout:         defaultDriverTimeout,
```

Comment:
If the driver timeout is still constant 120s and can't be changed, wouldn't it be safer if this would be added in the `driver` pkg. Atm I think you could still theoretically call driver pkg forgetting to pass in `InitConfig.Timeout` and then possibly bad things can happen as it doesn't seem to be validated.

Reply:
I added a check in GetDriver that the Timeout is set to something > 0.